export.py

```diff
@@ -368,11 +368,12 @@ def load_hf_model(model_path):
     config.dim = hf_model.config.hidden_size
     config.n_layers = hf_model.config.num_hidden_layers
     config.n_heads = hf_model.config.num_attention_heads
-    config.n_kv_heads = hf_model.config.num_attention_heads
+    config.n_kv_heads = hf_model.config.num_key_value_heads
```

Comment:
?

Reply:
For MHA model, the number of kv heads equals q heads. 
However, for GQA model like llama2-70b, tinyllama1.1B, the number of kv heads and q head are different.